book/special_variables.md

```diff
@@ -118,6 +118,37 @@ A list of directories which will be searched when using the `source`, `use`, or
 - [Module Path](./modules/using_modules.md#module-path)
 - [Configuration - `$NU_LIB_DIRS`](./configuration.md#nu-lib-dirs-constant)
 
+### `$env.NU_LOG_LEVEL`
+
+The [standard library](/book/standard_library.md) offers logging in `std/log`. The `NU_LOG_LEVEL` environment variable is used to define the log level being used.
+
+```nu
+nu -c '1 | print; use std/log; log debug 1111; 9 | print'
+# => 1
+# => 9
+
+nu -c '1 | print; use std/log; NU_LOG_LEVEL=debug log debug 1111; 9 | print'
+# => 1
+# => 2025-07-12T21:27:30.080|DBG|1111
+# => 9
+
+nu -c '1 | print; use std/log; $env.NU_LOG_LEVEL = "debug"; log debug 1111; 9 | print'
+# => 1
+# => 2025-07-12T21:27:57.888|DBG|1111
+# => 9
+```
+
+Note that `$env.NU_LIB_DIRS` is different from `nu --log-level`, which sets the log level for Nushell commands. It does not influence the `std/log` logging used in custom commands and scripts.
```

Comment:
I was think about whether there is a good, fitting qualifier for Nushell's own commands. In the end I chose not to add any qualifier, and only call the custom commands custom commands as a differentiation (like they are called in the docs).

```suggestion
Note that `$env.NU_LIB_DIRS` is different from `nu --log-level`, which sets the log level for native Nushell commands. It does not influence the `std/log` logging used in custom commands and scripts.
```

```suggestion
Note that `$env.NU_LIB_DIRS` is different from `nu --log-level`, which sets the log level for (native) Nushell commands. It does not influence the `std/log` logging used in custom commands and scripts.
```

Reply:
```suggestion
Note that `$env.NU_LOG_LEVEL` is different from `nu --log-level`, which sets the log level for Nushell commands. It does not influence the `std/log` logging used in custom commands and scripts.
```